packages/ui/src/components/Designer/RightSidebar/DetailView/index.tsx

```diff
@@ -96,23 +91,26 @@ const DetailView = (props: DetailViewProps) => {
     }
 
     let changes: ChangeSchemaItem[] = [];
-    for (let key in formSchema) {
+    for (const key in formSchema) {
       if (['id', 'content'].includes(key)) continue;
 
       let value = formSchema[key];
-      let changed = false;
-
-      if (['x', 'y'].includes(key)) {
```

Comment:
refactoring

Reply:
I'm seeing this now

![Screenshot 2024-08-09 at 05 56 17](https://github.com/user-attachments/assets/54b1f762-5425-44f7-bd0c-957bcbc04541)